documentation/docs/instructor.md

```diff
@@ -1,82 +1,99 @@
+### Shell Script
+The instance will run the script you provide here once it's created. Provide the appropriate shebang respective to the image type:
+
+Windows:
+```
+#ps1_sysnative 
+```
+
+Linux:
+```
+#!/bin/sh
+```
+
+### Install Packages (Linux Only)
+Install packages by inserting one package name per line. You can search for package names here: http://packages.ubuntu.com/.
+
+### Update Package Lists on Boot (Linux Only) 
+By selecting yes, it will update the list of available packages and their versions, but does not install or upgrade any packages.
+
+### Upgrade All Packages on Boot (Linux Only)
+By selecting yes, it will upgrade newer versions of the packages you have. 
+
+### Perform Reboot if Required After Package Install/Upgrade (Linux Only)
+By selecting yes, it will perform a reboot if packages are installed and upgraded.
+
+### Timezone (Linux Only)
+Set a different system timezone. 
+
 Example:
 ```
-American/New_York
+America/New_York
 ```
 
-## Hostname (Linux Only)
-Set the system hostname to be different than default value.
+### Hostname (Linux Only)
+Set a hostname that will replace the hostname generated by default.
+
+### Default User Password (Linux Only)
+Sets the password for default user (username is OS-dependant). Leave blank to disable.
+
+### Default User Public Key (Linux Only)
+SSH public key for default user (username is OS-dependant).
+
+Example:
+```
+ssh-rsa AAAB... user@host
+```
+
+### Create System Account for Student (Linux Only)
+Creates a user in the system based on student's username and a random password (will be provided to student).
+
+### Student User Sudo (Linux Only)
+Sets sudo access for the student's system user. Set to `'ALL = (ALL) NOPASSWD: ALL'` for unlimited sudo access.
+
+## Creating an Assignment
+Visit the Assignment List page by selecting the `Assignments` tab found in the header. Create a course by clicking the `Create Assignment` button in the top right corner and then providing the necessary information. Each assignment will be assigned to one course and will become available to the students upon the given start date. The assignment will close and all associate Environments will be terminated based on the set end date. Assignments can be updated or deleted on the Assignment List page. 
+
+`Course and Environment Definition must first be created in order for students to be assigned to an Assignment.` 
+
+## Entering Your Environment
+On the Assignment List page, click on an assignment title and then press the `Enter Your Environment` button (it will take a moment for the console to load). Once the console appears and reaches the log in screen, enter the username and password given in the blue box. The console can be in full screen mode by clicking the `Open Console in Full Screen Mode`. If there are multiple VMs, you can easily switch from one console to another by clicking on the desired VM found in the green box. 
+
+`Please note: CTRL+C/CTRL+V doesn't work in this console.`
```

Comment:
It's easier to say what works. Keyboard, Video, Mouse. Hence the name is KVM. So audio won't work.